packages/blockchain/src/index.ts

```diff
@@ -73,6 +73,12 @@ export interface BlockchainOptions {
    * provided, it defaults to `true`.
    */
   validateBlocks?: boolean
+
+  /**
+   * If a genesis block is present in the provided `db`, then this genesis block will be used.
```

Comment:
The "this" reads a bit confusing here, I would just repeat "the genesis block from `db` will be used (especially since there is another "this" below referencing `genesisBlock`, I actually would remove the "this" there as well for readability)